src/collections/_documentation/accounts/quotas/index.md

```diff
@@ -0,0 +1,216 @@
+
+2. **SDK sample rate**
+
+      If a sample rate is defined for the SDK, Sentry evaluates whether this event should be sent as a representative fraction of the sampled events. Setting a sample rate is documented for each SDK. For more information, see [Configuration, sampleRate]({%- link _documentation/error-reporting/configuration/index.md -%}#sample-rate).
+
+3. **Quota availability**
+
+      Events that exceed your quota are not sent. To add to your quota or review what happens when you exceed it, see [Increasing Quotas]({%- link _documentation/accounts/quotas/index.md -%}#increasing-quotas).
+
+4. **Event repetition**
+  -   If you have intervened to discard events with the same fingerprint, the event does not count toward your quota. For more information, see [Delete & Discard in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md  -%}#delete--discard) or [Inbound Filters]({%- link _documentation/accounts/quotas/index.md -%}#inbound-data-filters).
+  -   If the previous event was resolved, this event counts toward your quota because it may represent a regression in your code.
+  -   If you have intervened to ignore alerts about events with the same fingerprint, this event counts toward your quota because the event is still occurring. For more information, see [Inbound Filters]({%- link _documentation/accounts/quotas/index.md -%}#inbound-data-filters).
+
+5. **Spike protection**
+
+      Sentry’s spike protection prevents huge overages from consuming your event capacity. For more information, see [Spike Protection]({%- link _documentation/accounts/quotas/index.md -%}#spike-protection).
+
+In addition, depending on your project’s configuration and the plan you subscribe to, Sentry may also check:
+
+6. **Rate limit for the project**
+
+      If the rate limit for the project has been exceeded, and your subscription allows, the event will not be counted. For more information, see [Rate Limiting in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md -%}#4-rate-limiting) or  [Rate Limiting Projects]({%- link _documentation/accounts/quotas/index.md -%}#id1).
+
+7. **Inbound Filters**
+
+      If any inbound filter is set for this type of event, and your subscription allows, the event will not be counted. For more information, see [Inbound Filters in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md -%}#inbound-data-filters) or [Inbound Data Filter]({%- link _documentation/accounts/quotas/index.md -%}#inbound-data-filters).
+
+After these checks are processed, the event counts toward your quota. It is accepted into Sentry, where it persists and is stored.
```

Comment:
What might be worth mentioning: Events from disabled DSNs do **not** count against the quota.

Reply:
That is helpful. Yes, I will add. Thanks.

Reply:
Actually, we have it already at line 19 - "Before completing any of these evaluations, Sentry confirms that each event includes a valid DSN and project as well as whether the event can be parsed and contains valid fingerprint information; if any of these items are missing or incorrect, the event is rejected." 

I don't think it needs repeating?

Reply:
You’re right. If people jump right to this list,they might maybe miss it, but since “disabling” a DSN is quite straight-forward, it’s not worth repeating here.